<launch>
  <node name="area_navigation" pkg="area_navigation" type="area_navigation_node">
    <param name="gateway_detection_topic" value="/gateways_detected" />
    <param name="semantic_feature_detection_topic" value="/semantic_features_detected" />
    <param name="distance_monitor_topic" value="/monitored_distance" />
    <param name="heading_monitor_topic" value="/monitored_heading" />
    <param name="desired_heading_topic" value="/desired_heading" />
    <param name="desired_velocity_topic" value="/desired_velocity" />

    <param name="reset_distance_monitor_service" value="/reset_distance_monitor" />
    <param name="reset_heading_monitor_service" value="/reset_heading_monitor" />
    <param name="heading_control_switch_service" value="/heading_control_switch" />

    <param name="recovery_direction_threshold" value="1.0" />
    <param name="correction_direction_threshold" value="0.3" />

    <param name="controller_frequency" value="20" />
    <param name="velocity" value="0.5" />
  </node>
</launch>

